### ISSUE TITLE
Doesn't allow selection

### ISSUE THREAD

**Issue by BoredToHell69** [shubhamg0sai/virous#2]
This tool either needs updated or it's just poorly made

**Comment by DescobertasDigitais** [comment]
> This tool either needs updated or it's just poorly made

You're right, this tool does nothing but browse through some folders and then show a download link for a malicious file.
